README.md

```diff
@@ -1,284 +1,126 @@
-# `@cloudflare/next-on-pages`
+# ⚡▲ next-on-pages ▲⚡
 
-Reference:
+next-on-pages is a CLI tool that you can use to build and develop [Next.js](https://nextjs.org/) applications so that they can run on the [Cloudflare Pages](https://pages.cloudflare.com/) platform (and integrate with the various Cloudflare offerings such as KV stores, and Durable Objects).
 
-- [Blog](https://blog.cloudflare.com/next-on-pages)
-- [Documentation](https://developers.cloudflare.com/pages/framework-guides/deploy-a-nextjs-site/)
+This tool is a best-effort library implemented by the Cloudflare team and the community since Vercel doesn't and likely won't support Next.js running on Cloudflare officially. As such most but not all Next.js features are yet supported, see the [Supported Versions and Features document](./docs/supported.md) for more details.
 
 ## Quick Start
 
-1. `npx create-next-app@latest my-app`
+This section describes how to bundle and deploy a (new or existing) Next.js application and using next-on-pages.
 
-   Note that if you elect to use eslint, there are a couple of places you need to add return types to make the default template pass the pre-build checks.
+> Note: this readme uses `npm`, but nothing relies on it, you can instead use `yarn` or `pnpm` if you want.
```

Comment:
always mentioning `npm or yarn or pnpm` is a bit too verbose to me, I think it's much cleaner just to stick with `npm` and tell the dev that they can use the others if they want (converting from one to the other should be trivial anyways)

Reply:
I might need to rethink this statement, as using `pnpm` doesn't seem to work out of the box (see the last comments here: https://github.com/cloudflare/next-on-pages/issues/52)

Reply:
I've removed the note for now since `pnpm` doesn't work out of the box, we can leave it as is for now and add some info later if we want